docs/tutorials/cli.rst

```diff
@@ -198,9 +205,9 @@ To setup pseudolocalization add :conf:`pseudoLocale` in ``package.json``::
 :conf:`pseudoLocale` option can be any string 
 examples: :conf:`en-PL`, :conf:`pseudo-LOCALE`, :conf:`pseudolocalization` or :conf:`en-UK`
 
-PseudoLocale folder is automatically created based on configuration when running 
-``yarn extract`` command. Pseudolocalized text is created on  ``yarn compile`` command.
-The pseudolocalization is automatically created from default messages. 
+PseudoLocale string have to be in `locale` config as well. Otherwise no folder is going to be created.
+Pseudolocalized text is created on  ``yarn compile`` command.
+The pseudolocalization is automatically created from messages in order specified in :ref:`Preparing catalogs for production`.
 It can also be changed by setting translation in :conf:`message.json` into non-pseudolocalized text.
```

Comment:
Not sure what link was supposed to be here but I guess it should be sth like this:

```suggestion
It can also be changed by setting translation in catalog into non-pseudolocalized text.
```

Reply:
I think we should remove this sentence. It was supposed to show that users are able to edit message.json file manually. Considering that it would get removed after running compile then I do not find it useful.

Reply:
I have removed the sentence